<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientForm</class>
 <widget class="QWidget" name="PatientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="personalInfoPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Personal Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Full Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="dobLabel">
           <property name="text">
            <string>Date of Birth:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dobDateEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="genderLabel">
           <property name="text">
            <string>Gender:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="genderComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="contactLabel">
           <property name="text">
            <string>Contact Number:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="contactLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="emergencyContactLabel">
           <property name="text">
            <string>Emergency Contact:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="emergencyContactLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QTextEdit" name="addressTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="insurancePage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Insurance Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hasInsuranceRadio">
         <property name="text">
          <string>I have insurance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="insuranceGroupBox">
         <property name="title">
          <string>Insurance Details</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="providerLabel">
            <property name="text">
             <string>Insurance Provider:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="providerLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="memberIdLabel">
            <property name="text">
             <string>Member ID:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="memberIdLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="symptomsPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Symptoms</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="availableSymptomsLabel">
             <property name="text">
              <string>Available Symptoms:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="symptomList"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="severityLabel">
               <property name="text">
                <string>Severity (1-5):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="severitySpinBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="addSymptomButton">
               <property name="text">
                <string>Add Symptom</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="selectedSymptomsLabel">
             <property name="text">
              <string>Selected Symptoms:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="selectedSymptomList"/>
           </item>
           <item>
            <widget class="QPushButton" name="removeSymptomButton">
             <property name="text">
              <string>Remove Selected Symptom</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="medicalHistoryPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Medical History</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="medicalHistoryLabel">
         <property name="text">
          <string>Personal Medical History:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="medicalHistoryTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="familyHistoryLabel">
         <property name="text">
          <string>Family Medical History:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="familyHistoryTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="medicationsLabel">
         <property name="text">
          <string>Current Medications:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="medicationsTextEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="allergiesCheckBox">
         <property name="text">
          <string>I have medication allergies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="allergiesTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
